src/accidental.js

```diff
@@ -465,36 +465,42 @@ export class Accidental extends Modifier {
 
   // Render accidental onto canvas.
   draw() {
-    if (!this.context) {
+    const {
+      context,
+      type, position, note, index, cautionary,
+      x_shift, y_shift, width,
+      accidental, parenLeft, parenRight,
+      render_options: { font_scale },
+    } = this;
```

Comment:
Multi-line destructuring is okay. I guess I didn't understand how those variables were destructured from `this`, and now I do. It's definitely succinct. I don't have a strong opinion here, so you can leave it as is.